cvat/apps/annotation/ddln_spotter.py

```diff
@@ -51,11 +68,43 @@ def dump(file_object, annotations):
     from cvat.apps.annotation.validation import validate
     from tempfile import TemporaryDirectory
 
+    ddln_id = None
+    group = "msq"
+    map_file = "task_mapping.csv"
+    id_file_path = None
+
+    task = annotations.meta['task']['name']
+    task = task.split('_')[0]
+    curr_date = annotations.meta['dumped'][:10]
+
+    dir_prefix = "/home/django/share/incoming/{}/"
+    for dirp, dirn, files in os.walk(dir_prefix.format(task)):
+        if "spo_" in dirp:
```

Comment:
Is it possible that task directory might have multiple 'spo_' directories and should we fail in this case?
I think this case could be handled this way:
```python
def get_annotation_request_id(task_name):
	import pathlib
	base_dir = pathlib.Path("/home/django/share/incoming")
	task_dir = base_dir.joinpath(task_name)
	ddln_id_files = list(task_dir.glob("spo_*/ddln_id"))
	if len(ddln_id_files) != 1:
		return None
	return ddln_id_files[0].read_text().strip()
```